[benchmarks] Print % of compute compiled with TVM. Useful for debugging

bwasti opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#35 [benchmarks] Print % of compute compiled with TVM. Useful for debugging**

bwasti committed:
[benchmarks] Print % of compute compiled with TVM. Useful for debugging

wanchaol commented on test/benchmarks.py:
```diff
@@ -42,7 +42,15 @@ def benchmark(model, csv_file, input_fn=genImage, iters=100, warmup=10):
+        for p in prof.key_averages():
+          total_profiled_time += int(p.cpu_time)
+          if p.key == "TVM":
+            tvm_profiled_time = int(p.cpu_time)
```
I think you missed a `+` here.

bwasti replied:
done :)

bwasti committed:
Update on "[benchmarks] Print % of compute compiled with TVM. Useful for debugging"

[benchmarks] Print % of compute compiled with TVM. Useful for debugging

gh-metadata: pytorch tvm 35 gh/bwasti/22/head

wanchaol commented on test/benchmarks.py:
```diff
@@ -42,7 +42,15 @@ def benchmark(model, csv_file, input_fn=genImage, iters=100, warmup=10):
-        print("Done benchmarking TVM")
+        with torch.autograd.profiler.profile() as prof:
+          _ = trace_tvm(*inputs)
+        tvm_profiled_time = 0
```
hmmm shouldn't this whole block be under the indent of the `with` block? I thought `prof` will only be visible inside the block

bwasti replied:
nope, with doesn't create a scope
